### PR TITLE
GUACAMOLE-1291: Add Korean language support

### PR DESCRIPTION
**Pull request by OctopusET**
Add translation files for the webapp and the following extensions:

      - guacamole-auth-cas
      - guacamole-auth-duo
      - guacamole-auth-jdbc
      - guacamole-auth-openid
      - guacamole-auth-quickconnect
      - guacamole-auth-saml
      - guacamole-auth-totp

### CODE REVIEW

**Comment by OctopusET** [comment]
Hello, I just squashed every commit. 
Sorry for confusing commits. I try to make commit more meaningful, and reasonable. 

If co-author things should be removed I will do it.

**Comment by necouchman** [comment]
> If co-author things should be removed I will do it.

I'm in favor of keeping co-authors as it gives credit to people who've worked on the code, so I don't think you need to remove that, so long as it doesn't interfere with meaningful commit messages :-)

**Comment by OctopusET** [comment]
I want to let them co-authors. They made an effort for this translations.

**Comment by OctopusET** [comment]
Added missing translation from extensions.

**Comment by OctopusET** [comment]
Update ko.json to 1.3.0.
I'm sorry if updating commit is annoying.

@necouchman  is there anything should I do?

**Comment by necouchman** [comment]
@OctopusET: Force pushing the commit is fine, no issues with that. I'll try to review, soon, and make sure it looks okay.

File: extensions/guacamole-auth-radius/src/main/resources/translations/ko.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "DATA_SOURCE_RADIUS" : {
+        "NAME" : "RADIUS Backend"
+    },
+
+    "LOGIN" : {
+        "FIELD_HEADER_GUAC_RADIUS_STATE"              : "",
+        "FIELD_HEADER_RADIUSCHALLENGE"                : ""
+    }
+
+}
```
[the remark this answers is not in the document]
Comment (OctopusET) [reply extensions/guacamole-auth-radius/src/main/resources/translations/ko.json:12]: "Backend" can be translated, but writing in english is better. So, I will remove ko.json and restore guac.manifest.json

**Comment by necouchman** [comment]
> Then should I remove every lines that reference another lines?

I believe so, yes. @mike-jumper can you confirm?

> and should I remove lines like ("FIELD_OPTION_BACKSPACE_EMPTY" : "",)? Even these kind line should be empty?

Again, yes, I believe you can leave off empty lines.

**Comment by OctopusET** [comment]
@necouchman Okay I will remove all.

**Comment by OctopusET** [comment]
Removed unnecessary lines as @necouchman refers.

File: guacamole/src/main/webapp/translations/ko.json [hunk 2]
```diff
@@ -0,0 +1,843 @@
...
+        "HELP_CONNECTION_HISTORY" : "과거 연결 기록은 여기에 나열되며 열 헤더 (column header)를 클릭하여 정렬할 수 있습니다. 특정 기록을 검색하려면, 필터 문자열을 입력하고 \"검색\"을 클릭하십시오. 제공된 필터 문자열과 일치하는 기록만 나열됩니다.",
+
+        "INFO_CONNECTION_DURATION_UNKNOWN" : "--",
+        "INFO_NO_HISTORY"                  : "일치하는 기록 없음",
+
+        "TABLE_HEADER_SESSION_CONNECTION_NAME" : "연결 이름",
+        "TABLE_HEADER_SESSION_DURATION"        : "기간",
+        "TABLE_HEADER_SESSION_REMOTEHOST"      : "원격 호스트",
+        "TABLE_HEADER_SESSION_STARTDATE"       : "시작 시간",
+        "TABLE_HEADER_SESSION_USERNAME"        : "사용자 이름",
```
Comment (necouchman) [inline guacamole/src/main/webapp/translations/ko.json:745]: Extra comma needs to be removed.

File: guacamole/src/main/webapp/translations/ko.json [hunk 3]
```diff
@@ -0,0 +1,843 @@
...
+
+        "SECTION_HEADER_EDIT_CONNECTION" : "연결 편집",
+        "SECTION_HEADER_HISTORY"         : "사용 기록",
+        "SECTION_HEADER_PARAMETERS"      : "매개변수",
+        "TABLE_HEADER_HISTORY_USERNAME"   : "사용자 이름",
+        "TABLE_HEADER_HISTORY_START"      : "시작 시간",
+        "TABLE_HEADER_HISTORY_DURATION"   : "기간",
+        "TABLE_HEADER_HISTORY_REMOTEHOST" : "원격 호스트",
+
+        "TEXT_CONFIRM_DELETE"   : "연결을 삭제한 후에는 복원할 수 없습니다. 이 연결을 삭제하겠습니까?",
```
Comment (necouchman) [inline guacamole/src/main/webapp/translations/ko.json:214]: Extra comma needs to be removed.

**Review by necouchman — CHANGES_REQUESTED** [review]
Just a couple of extra commas that need to be removed in the main translation, and I think it'll be good.

**Comment by OctopusET** [comment]
@necouchman commas removed. Sorry I didn't notice them.

**Review by necouchman — APPROVED** [review]
Okay, good as far as I can tell. Merge here momentarily.

**Comment by OctopusET** [comment]
Thank you for review.

**Pull request merged**